Dockerfile

```diff
@@ -21,53 +21,62 @@ RUN --mount=type=cache,target=/go/pkg/mod/ \
 # source code into the container.
 RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=bind,target=. \
-<<EOF
+    <<EOF
     export VERSION_PKG_PATH="go.autokitteh.dev/autokitteh/internal/version"
     export TIMESTAMP="$(date -u "+%Y-%m-%dT%H:%MZ")"
-    export LDFLAGS="-X "${VERSION_PKG_PATH}.Version=$(cat .version || echo)" -X "${VERSION_PKG_PATH}.Time=${TIMESTAMP}" -X "${VERSION_PKG_PATH}.Commit=$(cat .commit || echo)""
+    export LDFLAGS="-s -w -X "${VERSION_PKG_PATH}.Version=$(cat .version || echo)" -X "${VERSION_PKG_PATH}.Time=${TIMESTAMP}" -X "${VERSION_PKG_PATH}.Commit=$(cat .commit || echo)""
     CGO_ENABLED=0 go build -o /bin/ak -ldflags="${LDFLAGS}" ./cmd/ak
 EOF
 
-################################################################################
-# Create a new stage for running the application that contains the minimal
-# runtime dependencies for the application.
+FROM python:3.12 AS python_deps
 
-FROM python:3.12-alpine3.20 AS final
+WORKDIR /app
 
-# Install any runtime dependencies 
-RUN --mount=type=cache,target=/var/cache/apk \
-    apk --update add \
-    ca-certificates \
-    tzdata \
-    && \
-    update-ca-certificates
+# RUN --mount=type=cache,target=/var/cache/apk \
+#     apt-get \
+#     g++ libstdc++ \
+#     ca-certificates \
+#     tzdata \
+#     && \
+#     update-ca-certificates
 
 COPY ./scripts/gcc-on-arm.sh /tmp
 RUN /tmp/gcc-on-arm.sh
 RUN rm /tmp/gcc-on-arm.sh
 
+
+RUN python3 -m venv env
+COPY ./runtimes/pythonrt/requirements.txt requirements.txt
+ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1
+ENV GRPC_PYTHON_BUILD_SYSTEM_ZLIB=1
+RUN /app/env/bin/python3 -m pip install --no-cache-dir -r requirements.txt
+
+################################################################################
+# Create a new stage for running the application that contains the minimal
+# runtime dependencies for the application.
+
+FROM python:3.12-alpine3.20 AS final
+
+
 # Create a non-privileged user 
 # See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
 ARG UID=10001
 RUN adduser \
     --disabled-password \
     --gecos "" \
-    --home "/nonexistent" \
+    --home "/usr/src/autokitteh" \
```

Comment:
why not in regular place for user's home? e.g. /home/someuser?

Reply:
it's not a real user and it has no shell
so I'm not sure it matters

but let me know if its critical to you and I'll try and change it